scripts/zoho_crm.py

```diff
@@ -0,0 +1,583 @@
+import os
+import django
+import sys
+import logging
+import json
+from typing import List, Dict, Optional, Any, Callable
+from datetime import datetime, timedelta
+import requests
+import json
+import csv
+from collections import defaultdict
+from typing import List, Dict, Optional, Any, Tuple
+import zipfile
+import tempfile
+
+from django.db.models import Model
+
+project_path = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.append(project_path)
+
+# Set up Django environment
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "daras_ai_v2.settings")
+django.setup()
+
+from app_users.models import AppUserTransaction, PaymentProvider, TransactionReason
+from daras_ai_v2 import settings
+
+ZOHO_CONTACT_API = "https://www.zohoapis.com/crm/v2/Contacts"
+ZOHO_DEAL_API = "https://www.zohoapis.com/crm/v7/Deals"
+ZOHO_HEADERS = {"Authorization": f"Bearer {settings.ZOHO_AUTH_CODE}"}
```

Comment:
generate this inside a method like we do in `azure_image_moderation` so that settings can be loaded at runtime if needed